src/zappy_gui_src/Core/SFML/Map/Map.cpp

```diff
@@ -48,21 +51,65 @@ void Map::_updateMoveMap(sf::Vector2f &moveMap)
 void Map::display()
```

Comment:
I don't understand what this function do, many call to clock with -1 index and we don't know why.
Can you please split in understandable function, will appreciate that